app.js

```diff
@@ -1,22 +1,35 @@
 const express = require("express")
 const app = express()
-const { 
-    getTopics,
-    getArticles
+const {
+  getTopics,
+  getArticles,
+  getArticleByID
 } = require("./controllers")
 
 app.get("/api/topics", getTopics)
 app.get("/api/articles", getArticles)
+app.get("/api/articles/:article_id", getArticleByID)
+
+app.use((req, res, next) => {
+  res.status(404).send({msg: "Path not found"})
+})
 
 app.use((err, req, res, next) => {
-    if (err.status === 404) {
-      res.status(err.status).send({ message: err.message });
-    } else next(err);
+  if (err.status === 404) {
+    res.status(err.status).send({ message: err.msg });
+  } else next(err);
+});
+
+
+app.use((err, request, response, next) => {
+  if (err.code === "22P02") {
+    response.status(400).send({ message: "Bad Request" });
+  } else next(err);
 });
 
```

Comment:
Could this be put into a separate file to avoid confusion in your app?

Reply:
I'm not sure how to place this in another file and still get it to work